travis.sh

```diff
@@ -2,8 +2,8 @@
 #set -o errexit
 
 cd definitions && \
-#yarn install && \
-#yarn test && \
+yarn install && \
+yarn test && \
```

Comment:
Change to `yarn run test-quick`, we don't need it to build. Also consider changing it to `yarn run lint`, if you're just looking to run eslint without running jest.

Reply:
@GAntoine, tbh, this PR made me a little uneasy because the root of the problem is that the travis script does not run what the Contributing guide tells contributors to run (ie `run_def_tests.sh`). Why not have it do that instead? Any discrepancy between the class of errors/problems that Travis alerts contributors to and the errors/problems they find during development will cause confusion.

Reply:
For the sake of consistency, I agree with the change and I'll merge this PR. In the future I plan on rewriting the contributing guide, and move away from `run_def_tests.sh`.